ansible/roles/rancher/tasks/main.yml

```diff
@@ -1,12 +1,13 @@
 ---
 - name: Probe Helm Repos
-  command: helm repo list
+  command: helm repo list --output json
   changed_when: false
   run_once: true
+  ignore_errors: true
```

Comment:
Why do we want to ignore errors?

Reply:
My fault for leaving this for so long, I believe it was because `--output json` actually returned an error code when it was empty. I'll test again.